WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadService.java

```diff
@@ -318,6 +318,20 @@ public static boolean isPostUploading(PostModel post) {
         return post != null && PostUploadHandler.isPostUploading(post);
     }
 
+    public static void abortPostAndRelatedMediaUpload(PostModel post) {
```

Comment:
Could we refactor this to call `cancelQueuedPostUpload()` internally and drop some code duplication?

Also, what do you think of calling it `cancelQueuedPostUploadAndRelatedMedia()`, for a bit more consistency with that (similarly-behaving) method?